src/Manager/PostManager.php

```diff
@@ -180,7 +180,21 @@ public function storeAttachments(Post $post)
      */
     private function downloadFile($url, $basename, $dir)
     {
-        $fileContents = @file_get_contents($url);
+    	$arrContextOptions = [];
+
+    	$config = $this->grav['config']->get('plugins.social-feed');
```

Comment:
Please add 2 spaces for indentation.

Reply:
Hi 
I updated the file indentation. Can you please check that?
https://github.com/Thoomyy/grav-plugin-social-feed/commit/88f05eb7d40269345d3179ed8c8ff1a9e8973223